hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieInstantTimeGenerator.java

```diff
@@ -129,7 +129,7 @@ public static String getInstantForDateString(String dateString) {
   }
 
   private static TemporalAccessor convertDateToTemporalAccessor(Date d) {
-    return d.toInstant().atZone(ZoneId.systemDefault()).toLocalDateTime();
+    return d.toInstant().atZone(getZoneId()).toLocalDateTime();
   }
 
```

Comment:
Can we supplement some UTs for `parseDateFromInstantTime` and `convertDateToTemporalAccessor` ?

Reply:
> Can we supplement some UTs for `parseDateFromInstantTime` and `convertDateToTemporalAccessor` ?

I would be happy to do it.

Reply:
> convertDateToTemporalAccessor

I added two UTs: `testFormatDateWithCommitTimeZone` and `testInstantDateParsingWithCommitTimeZone`,  `testInstantDateParsingWithCommitTimeZone` is used to test the correctness of the `HoodieInstantTimeGenerator#convertDateToTemporalAccessor()` via `formatDate()`

Reply:
> Can we supplement some UTs for `parseDateFromInstantTime` and `convertDateToTemporalAccessor` ?

And in the TestHoodieActiveTimeline.java, there are many UTs related to DateParsing, such as: 
- `testInvalidInstantDateParsing` 
- `testMillisGranularityInstantDateParsing` 
etc.